src/utils/string/index.ts

```diff
@@ -0,0 +1,23 @@
+import { DurationInputArg1, duration } from "moment";
+
+export const msToDuration = (ms: DurationInputArg1): string => {
```

Comment:
Weird that moment doesn't have a built-in way to format durations. Since this is the only place we're using it currently, does it make sense to use a different time library?

Reply:
i second this... moment is a HUGE dependency and it does not look like we're accessing its full potential here